src/attributes/components/AttributeValues/AttributeValues.tsx

```diff
@@ -182,7 +182,7 @@ const AttributeValues: React.FC<AttributeValuesProps> = ({
                       style={
                         value?.file
                           ? { backgroundImage: `url(${value.file.url})` }
-                          : { backgroundColor: value.value }
+                          : { backgroundColor: value?.value }
```

Comment:
**suggestion**:  this is not really type-safe, a question mark means you are sure what to expect. value can be undefined and when it does, you are ending up with backgroundColor set to undefined. I know that was within the code, but we can improve. If you have time, try to think to cover that scenario. Perhaps if there is no value, you can skip styling?